### DISCUSSION TITLE
In 9.5.0.x the console button has changes to consoles and has submenus

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#12907] [Q&A] [answered]
In 9.5.0.x the console button has changes to consoles and has submenus (console, and manage filesystem).

Imo manage filesystem shouldn't be under console(s) but on the mainmenu or in settings.

**Comment by sfromis** [comment, marked as answer]
Well, many ways of slicing it. One was chosen, where the consoles menu also can house Berry scripting console and the partition manager. The change is older than 9.5. The logic of putting file system management in the "Configuration" menu is not strong, either.

**Reply by jeroenst** [reply]
Imo a filemanager is not a thing that should be under a console(s) menu.. Maybe the name of consoles should be renamed to advanced or something else.
